app/notification.go

```diff
@@ -569,191 +568,6 @@ func (a *App) GetMessageForNotification(post *model.Post, translateFunc i18n.Tra
 	}
 }
 
-func (a *App) sendPushNotification(post *model.Post, user *model.User, channel *model.Channel, senderName, channelName string, wasMentioned bool) *model.AppError {
```

Comment:
I moved all the push notifications related stuff to the push_notification.go file.